docs/Quick-Start-Guide/Architecture.md

```diff
@@ -112,7 +110,7 @@ volume create: test-volume: success: please start the volume to access data
 
 ​4. **Striped Glusterfs Volume** - Consider a large file being stored in
```

Comment:
I think, this Volume type is not supported now

Reply:
Should I raise an issue and use a PR to make the changes?